src/lib/snark_worker/standalone/run_snark_worker.ml

```diff
@@ -19,8 +20,14 @@ let command =
      in
      fun () ->
        let open Async in
-       let constraint_constants =
-         Genesis_constants.Compiled.constraint_constants
+       let open Deferred.Let_syntax in
+       let%bind constraint_constants, proof_level =
+         let logger = Logger.create () in
```

Comment:
This is a correct usage of logger, as it's a daemon-like runner rather than a CLI command.

Reply:
This is inaccurate, this really is a CLI command. That said, I added it for my own debugging purposes, so if you want to pollute the output then go for it.

Reply:
For a command used for debugging some log output should be helpful